docs/checking-efk-health.md

```diff
@@ -1,123 +0,0 @@
-# Checking EFK Health
```

Comment:
i feel like this is still relevant as well

Reply:
I though i reviewed this content and the checks didnt really apply and/or out of date.  Should likely be move to the various "troubleshooting" sections in EO or CLO